src/wire/ipv4.rs

```diff
@@ -75,6 +75,55 @@ impl fmt::Display for Address {
     }
 }
 
+/// An IPv4 CIDR containing an IPv4 address with a prefix lenght.
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Copy, Default)]
+pub struct Cidr {
+    address: Address,
```

Comment:
This is a really odd case, because so far every other data structure in `wire` has all fields `pub`. It's even in the module documentation. Thoughts?

Reply:
The idea was to prevent the creation of inconsistent values, instead of using `is_valid` method, since CIDRs have an invariant that needs to be enforced.